Release 4.10.0 (#160)
blackbaud/skyux-lookup: CHANGELOG.md

# 4.10.0 (2021-03-17)

> - Addded the abilitly to include an add button to the lookup component results dropdown. [#158](https://github.com/blackbaud/skyux-lookup/pull/158)

Reviewer: ```suggestion
- Added the option of including an add button for the lookup component results dropdown. [#158](https://github.com/blackbaud/skyux-lookup/pull/158)
```
